pkg/jx/cmd/gke/helper.go

```diff
@@ -1,55 +1,28 @@
 package gke
 
-func GetGoogleZones() []string {
-
-	return []string{
-		"asia-northeast1-a",
-		"asia-northeast1-b",
-		"asia-northeast1-c",
-		"asia-southeast1-a",
-		"asia-southeast1-b",
-		"asia-east1-a",
-		"asia-east1-b",
-		"asia-east1-c",
-
-		"australia-southeast1-a",
-		"australia-southeast1-b",
-		"australia-southeast1-c",
+import (
+	"fmt"
+	"os/exec"
+	"sort"
+	"strings"
+)
 
-		"europe-west1-b",
-		"europe-west1-c",
-		"europe-west1-d",
-		"europe-west2-a",
-		"europe-west2-b",
-		"europe-west2-c",
-		"europe-west3-a",
-		"europe-west3-b",
-		"europe-west3-c",
-		"europe-west4-b",
-		"europe-west4-c",
-
-		"northamerica-northeast1-a",
-		"northamerica-northeast1-b",
-		"northamerica-northeast1-c",
-
-		"southamerica-east1-a",
-		"southamerica-east1-a",
-		"southamerica-east1-a",
+func GetGoogleZones() []string {
+	var zones []string
+	out, err := exec.Command("gcloud", "compute", "zones", "list").Output()
+	if err != nil {
+        fmt.Println("error occured")
```

Comment:
let's return the error if we cannot list the zones as we wont be able to proceed